cms/djangoapps/contentstore/toggles.py

```diff
@@ -62,3 +62,21 @@ def bypass_olx_failure_enabled():
     Check if bypass is enabled for course olx validation errors.
     """
     return BYPASS_OLX_FAILURE.is_enabled()
+
+
+# .. toggle_name: FEATURES['ENABLE_EXAM_SETTINGS_HTML_VIEW']
```

Comment:
ENABLE_PROCTORING_EXAM_SETTINGS_HTML_VIEW would be more clear.

Reply:
@asadazam93 we used an existing feature flag to refactor some code - this PR didn't add this feature flag.